packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2306,24 +2307,39 @@ export const handlers = [
   }),
   http.get('*/monitor/services', () => {
```

Comment:
Can we build a factory for this and not type it anymore

Reply:
added new factory for services